apps/portal/prisma/schema.prisma

```diff
@@ -76,6 +80,64 @@ enum TodoStatus {
 // across all models in this file.
 // End of Resumes project models.
 
+model ResumesResume {
+    id             String           @id @default(cuid())
+    userId         String
+    title          String           @db.Text
+    additionalInfo String           @db.Text
+    // TODO: role
+    // TODO: experience
+    // TODO: location
+    url            String
+    createdAt      DateTime         @default(now())
+    updatedAt      DateTime         @updatedAt
+    user           User             @relation(fields: [userId], references: [id], onDelete: Cascade)
+    stars          ResumesStar[]
+    comments       ResumesComment[]
+}
+
+enum Section {
+    GENERAL
+    EDUCATION
+    EXPERIENCE
+    PROJECTS
+    SKILLS
+}
+
+model ResumesStar {
+    id        String        @id @default(cuid())
+    resumeId  String
+    userId    String
+    createdAt DateTime      @default(now())
+    updatedAt DateTime      @updatedAt
```

Comment:
If the user un-stars a resume, are we just deleting the star? If so would we still need an `updatedAt` field?